util/file_reader_writer.h

```diff
@@ -317,9 +344,12 @@ class FilePrefetchBuffer {
   bool track_min_offset_;
 };
 
+// Returns a WritableFile.
```

Comment:
It will be nice it can be further improved. For now it seems to be obvious to me, as argument `result` is a `WriteableFile`.